js/interactive-guides/modules/tabbed-editor.js

```diff
@@ -282,6 +286,24 @@ var tabbedEditor = (function() {
 
             // Fill in the editors for this Tabbed Editor in different tabs
             var editors = content.editorList || [];
+
+            // Determine the width of the tabs based on the bootstrapColSize
+            // that the tabbed editor is given.
+            var numNonActiveEditors = editors.length - 1;
+            if (numNonActiveEditors < 2) {     
+                numNonActiveEditors = 1;
+                thisTabbedEditor.activeTabSize = '100%';
+            } else {
+                thisTabbedEditor.activeTabSize = '50%';
+            }
+            if (content.bootstrapColSize === "col-sm-12") {
+                // full page width tabbed editor
+                thisTabbedEditor.tabSize = Math.floor(100/numNonActiveEditors) + '%';
```

Comment:
I think we should specify that tabSize is for non-active tabs, or at least have a comment specifying that it will only affect the non-active tabs.